src/test/java/SignUp.java

```diff
@@ -0,0 +1,77 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.*;
+
+
+import java.time.format.DateTimeFormatter;
+import java.util.concurrent.TimeUnit;
+
+
+
+
+public class SignUp {
+
+
+    @Test
+    public static void signUpSucessTest() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(("yyyyMMddHHmmss"));
+        LocalDateTime myDate = LocalDateTime.now();
+        String localTimeString = formatter.format(myDate);
+        String date1= localTimeString.toString();
+        String com = new String("@gmail.com");
+        String email = new String(date1.concat(com));
```

Comment:
you use com variable only once when declarate email variable. Are you sure that you need to store a com as a separate variable? you can write it in one line: String email = date1 + "@gmail.com";